### PR TITLE
feat: Adding annotation option to frontend service

### PR DESCRIPTION
**Pull request by kuntalkumarbasu**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-kubernetes/datahub/charts/datahub-frontend/templates/service.yaml [hunk 1]
```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "datahub-frontend.fullname" . }}
   labels:
     {{- include "datahub-frontend.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
```
Comment (dexter-mh-lee) [inline datahub-kubernetes/datahub/charts/datahub-frontend/templates/service.yaml:7]: Can we add to GMS service.yaml as well?

Comment (kuntalkumarbasu) [reply datahub-kubernetes/datahub/charts/datahub-frontend/templates/service.yaml:7]: @dexter-mh-lee definitely I will send another PR for that once this is approved.

Comment (kuntalkumarbasu) [reply datahub-kubernetes/datahub/charts/datahub-frontend/templates/service.yaml:7]: @dexter-mh-lee pushed another commit for gms service

**Review by dexter-mh-lee — APPROVED** [review]
LGTM!

**Review by shirshanka — APPROVED** [review]
LGTM. Thanks Zack

**Pull request merged**